content/docs/releases/README.md

```diff
@@ -292,6 +292,7 @@ are no longer supported.
 
 | Release  | Release Date |     EOL      | Compatible Kubernetes versions | Compatible OpenShift versions |
 |----------|:------------:|:------------:|:------------------------------:|:-----------------------------:|
+| [1.15][] | Jun 05, 2024 | Feb 03, 2025 |          1.25 → 1.32           |          4.12 → 4.16          |
```

Comment:
Seems about right. I guess 1.24 -> 1.25 is because of some new requirement in cert-manager